features/step_definitions/front_office/renewal_steps.rb

```diff
@@ -435,10 +375,11 @@
   @renewals_app.company_name_page.submit
   @renewals_app.manual_address_page.submit(
     house_number: "1",
-    address_line_one: "Test lane",
-    address_line_two: "Testville",
-    city: "Teston",
-    country: "Testopia"
+    address_line_one: "Via poerio",
+    address_line_two: "Via poerio",
+    postcode: "00152", # required, as tests currently fail without a postcode
```

Comment:
Shall we raise a ticket for this?

Reply:
Currently it's covered in the comments in RUBY-729. I'm happy to split it out as a separate bug if preferred though.

Reply:
Let's make a TBD and discuss at the next meeting if we should work on it? :)

Reply:
OK - will do!

Reply:
I've now raised this as bug RUBY-769.